include/pros/screen.h

```diff
@@ -0,0 +1,360 @@
+/**
+ * \file screen.h
+ *
+ * Brain screen display and touch functions.
+ *
+ * Contains user calls to the v5 screen for touching and displaying graphics.
+ *
+ * \copyright (c) 2017, Purdue University ACM SIGBots.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef _PROS_SCREEN_H_
+#define _PROS_SCREEN_H_
+
+#include <stdarg.h>   
+#include <stdbool.h>  
+#define _GNU_SOURCE
+#include <stdio.h>  
+#undef _GNU_SOURCE
+#include <stdint.h>
+
+#include "pros/colors.h"     // c color macros
+
+#ifdef __cplusplus
+extern "C" {
+namespace pros {
+#endif
+
+/**
+ * Different font sizes that can be used in printing text.
+ * Also can determine if the medium/large text is centered or not.
+ * 
+ *  E_TEXT_SMALL - Small text font size
+ *  E_TEXT_MEDIUM - Normal/Medium text font size
+ *  E_TEXT_LARGE - Large text font size
+ *  E_TEXT_MEDIUM_CENTER - Medium centered text
+ *  E_TEXT_LARGE_CENTER - Large centered text
+ * 
+ */
+typedef enum {
+    E_TEXT_SMALL = 0, 
+    E_TEXT_MEDIUM,
+    E_TEXT_LARGE,
+    E_TEXT_MEDIUM_CENTER,
+    E_TEXT_LARGE_CENTER
+} text_format_e_t;
+
+/**
+ * Can be used to determine what the current touch status is for the touchscreen.
+ *  
+ *  E_TOUCH_PRESSED - Last interaction with screen was a quick press (not released yet)
+ *  E_TOUCH_RELEASED - Last interaction with screen was a release
+ *  E_TOUCH_HELD - User is holding screen down (not released or quick press)
+ * 
+ */
+
+typedef enum {
+    E_TOUCH_RELEASED = 0,
+    E_TOUCH_PRESSED,
+    E_TOUCH_HELD
+} last_touch_e_t;
+
+struct TouchStatus {
+    int16_t screen_last_x;
+    int16_t screen_last_y;
+    last_touch_e_t screen_touch_status;
+};
+
+#ifdef PROS_USE_SIMPLE_NAMES
```

Comment:
Seems like the simple names API was not implemented properly. Git did not push changes I made for some reason.